jax/_src/scipy/stats/_core.py

```diff
@@ -23,19 +23,30 @@
 from jax import jit
 from jax._src import dtypes
 from jax._src.api import vmap
-from jax._src.numpy.util import check_arraylike, implements, promote_args_inexact
+from jax._src.numpy.util import check_arraylike, promote_args_inexact
 from jax._src.typing import ArrayLike, Array
 from jax._src.util import canonicalize_axis
 
-import scipy
 
 ModeResult = namedtuple('ModeResult', ('mode', 'count'))
 
-@implements(scipy.stats.mode, lax_description="""\
-Currently the only supported nan_policy is 'propagate'
-""")
 @partial(jit, static_argnames=['axis', 'nan_policy', 'keepdims'])
 def mode(a: ArrayLike, axis: int | None = 0, nan_policy: str = "propagate", keepdims: bool = False) -> ModeResult:
+  """Compute the mode along an axis of an array.
+
+  JAX implementation of :func:`scipy.stats.mode`.
+
+  Args:
+    a: arraylike
```

Comment:
In general, should these reference `ArrayLike` ? Same everywhere.

Reply:
As discussed offline: this will require a fix to sphinx autodoc that we don't yet know how to do. We can address that separately.